src/transformers/models/fastspeech2_conformer/modeling_fastspeech2_conformer.py

```diff
@@ -1676,6 +1681,6 @@ def forward(
         waveform = self.vocoder(spectrogram)
 
         if not return_dict:
-            return model_outputs + (waveform,)
```

Comment:
Technically a breaking change, but more aligned with FastSpeech2ConformerWithHifiGanOutput's docstrings

Reply:
As above - docstrings should reflect code. If there's an inconsistency, it's either a bug in the code or the docstrings are wrong. 

Two questions: 
* Were people ever using this model as stand-alone? 
* Why does the model inherit from `PreTrainedModel` rather than its own `FastConformerPreTrainedModel` class?

Reply:
> Were people ever using this model as stand-alone?

Yes, it's a [model](https://huggingface.co/espnet/fastspeech2_conformer_with_hifigan) that directly outputs waveforms instead of spectrograms. However, downloads are very low (500ish last month).


> Why does the model inherit from PreTrainedModel rather than its own FastConformerPreTrainedModel class?

Not sure about this one, I can't find any info about in on the original PR!